xml/operations-tutorials-quickstart_guide.xml

```diff
@@ -390,9 +390,9 @@ ansible-playbook -i hosts/verb_hosts site.yml</screen>
        The <filename>site.yml</filename> playbook installs packages, starts
        services, configures network interface settings, sets iptables firewall
        rules, and more. Upon successful completion of this playbook, your
-       &kw-hos; cloud will be in place and in a running state. This playbook
-       can take up to six hours to complete, so sit back, relax, and pat
-       yourself on the back!
+       &kw-hos; will be in place and in a running state. This playbook can take
+       up to six hours to complete, so sit back, relax, and pat yourself on the 
+       back!
```

Comment:
Please remove the final half-sentence... that's a bit too much WTF-ery: `This playbook can take up to six hours to complete.`